README.md

```diff
@@ -41,6 +41,7 @@ This library currently supports the following cryptocurrencies and address forma
  - BCH (base58check and cashAddr; decodes to cashAddr)
  - BNB (bech32)
  - XLM (ed25519 public key)
+ - XMR (base58xmr public key)
```

Comment:
Can you provide reference link so that we can verify?

Reply:
@makoto 
Monero Base58 Docs https://monerodocs.org/cryptography/base58/
Monero Address Generate:  https://xmr.llcoins.net/